src/con_utils.ts

```diff
@@ -93,6 +96,16 @@ export function parseConnectArguments(
   };
 }
 
+function stashPath(projectDir: string): string {
+  const projectPath = fs.realpathSync(projectDir);
```

Comment:
```suggestion
  let projectPath = fs.realpathSync(projectDir);
  if (process.platform === "win32" && !projectPath.startsWith("\\\\")) {
    projectPath = "\\\\?\\" + projectPath;
  }
```

Reply:
@jaclarke Should this be implemented in https://github.com/edgedb/edgedb-python/pull/180 too?

Reply:
Yes, it looks like `os.path.realpath` in python behaves the same as `fs.realpath` in node.js

Reply:
Please suggest the change in that PR too, so that Paul does not miss this